.github/workflows/ci.yaml

```diff
@@ -0,0 +1,97 @@
+# yamllint disable rule:line-length
```

Comment:
Let's drop these comments as the are not useful here.

Reply:
Well, they're quite useful here if you happen to need to validate your work with `yamllint`. 

Line-length is an error for yamllint, by default.

```
$ yamllint .github/workflows/ci.yaml 
.github/workflows/ci.yaml
  4:1       warning  missing document start "---"  (document-start)
  6:1       warning  truthy value should be one of [false, true]  (truthy)
  51:81     error    line too long (105 > 80 characters)  (line-length)
  71:81     error    line too long (126 > 80 characters)  (line-length)
  73:81     error    line too long (95 > 80 characters)  (line-length)
  80:81     error    line too long (134 > 80 characters)  (line-length)
  87:81     error    line too long (102 > 80 characters)  (line-length)
```

Reply:
I've removed `disable rule:braces` though. It isn't used.

Reply:
> they're quite useful here if you happen to need to validate your work with `yamllint`.

I know, but we do validate with `yamllint`, right? It suggests we do, which is misleading.

If in the future we add that validation I'd like us to disable validation rules when needed, not because it was already there.

Reply:
It might be about time to start to validate these files with `yamllint` if you happen to edit them, what do you think?

I've edited YAML too many times to know it is very easy to make a bad edit. And what do you think will happen in this case? Nothing. No failure. Just nothing starts. No report from GitHub. No error. No warning. Nada. It's very annoying, and `yamllint` indispensable here. It saves you a lot of trouble by telling you right away if you made a mistake, without any need to scratch your head.

Reply:
There's no need to do this on CI: if CI starts, this means the file is OK. But right before a commit or a push, this is where `yamllint` brings the most value.

Reply:
And, while w're here, it is actually very easy to run it on CI: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5268/commits/5e0a977e0bfb9dab1e4ce3599bd19791ac8c6bd0

Reply:
Resolving this since the YAML check has been merged.